bot/cogs/moderation/scheduler.py

```diff
@@ -78,6 +84,9 @@ def mod_log(self) -> ModLog:
         icon = utils.INFRACTION_ICONS[infr_type][0]
         reason = infraction["reason"]
         expiry = infraction["expires_at"]
+        id_ = infraction['id']
```

Comment:
Should this be a double quote?

Reply:
There's not an explicit style set though majority prefer double IIRC. There was talks of adding a flake8 plugin for it and if that goes through then this will be taken care of then.